### ISSUE TITLE
Hapi / Handlebars: languageHeaderField: 'accept-language' not working

### ISSUE THREAD

**Issue by epochslee** [funktionswerk/hapi-i18n#22]
I have this plugin registered with 'accept-language'. The README suggests `language` but that does not detect the browser's language.

```
{
    plugin: require('hapi-i18n'),
    options: {
        locales: ['en', 'es'],
        directory: __dirname + '/locales',
        languageHeaderField: 'accept-language'
    }
}
```

How can I get my app to automatically use the right locale depending on the user's language set in their browser?

**Comment by kaywolter** [comment]
Browser language was not supported, yet. I have opened PR #23 to add it. Can you please check if the updates work on your end? I will merge the changes into master and publish a new release then.

**Comment by kaywolter** [comment]
PR is merged and new version 2.1.0 published to npm.
